lib/1.4/dml-builtins.dml

```diff
@@ -32,7 +33,7 @@ param dml_1_4 = true;
 
 constant unmapped_offset = 0xffff_ffff_ffff_ffff;
 
-extern void *const NULL;
+param NULL auto;
```

Comment:
why isn't this needed in 1.2?

Interesting solution to use an autoparam for NULL. It means that you can't use NULL in early evaluation, but not much happens between extern and param. I guess this gives a regression in some weird corner, perhaps `typedef typeof(NULL) null_t;` goes from valid to invalid. I don't mind.